app/components/spotlight/select_image_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render Blacklight::System::ModalComponent.new do |component| %>
+  <% component.with_title { t('spotlight.pages.form.select_image') } %>
+  <% component.with_body do %>
+    <div class='p-3 m-2'>
+      <input type="hidden" name="select_image_region">
+      <p class="instructions"><%= help_text.html_safe %></p>
+      <div class="mx-3" id="resource_block_iiif_cropper" data-behavior="iiif-cropper" data-cropper-key="select_image_<%=@block_item_id%>" data-index-id="<%=@index_id%>" data-crop-width="120" data-crop-height="120">
```

Comment:
Can the ID `resource_block_iiif_cropper` be removed? I didn't see it used later.

Reply:
The id is required by the cropping code, specifically in renderCropperMap in crop.js: this.cropArea.attr('id').  So we will need to keep the id for the code as it currently stands.

Reply:
Yes. I see now, thank you for pointing that out.